entities/entities.go

```diff
@@ -83,3 +84,12 @@ type FunctionAuth struct {
 	Token string
 	Type  AuthenticationType
 }
+
+type Deployment struct {
+	ID                  ID
+	UserID              string
+	Name                string
+	Location            string
```

Comment:
The server only returns the `ID`, `Name` and `CreatedAt` fields

Reply:
addressed